pkgs/top-level/all-packages.nix

```diff
@@ -12468,7 +12468,9 @@ let
 
   sxhkd = callPackage ../applications/window-managers/sxhkd { };
 
-  msmtp = callPackage ../applications/networking/msmtp { };
+  msmtp = callPackage ../applications/networking/msmtp {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
I don't think that this fails on non-Apple, but I have currently no way to test this. Anybody else?

I've modeled my changes on what is already in nixpkgs: Inheriting Darwin frameworks happens multiple times in `all-packages.nix` and since the framework is only a build input on Darwin, it should not matter on non-Apple systems.